README.md

```diff
@@ -6,39 +6,163 @@ Please edit this README.md file with your answers to the worksheet questions.
+|TRUE |FALSE|TRUE               |TRUE               |
+|TRUE |TRUE |FALSE              |FALSE              |
+
 ### c
 
+| *A* | *B* | *C* |(*A* AND *B*) OR (*A* AND *C*) |*A* AND (*B* OR *C*)           |
+|:---:|:---:|:---:|:-----------------------------:|:-----------------------------:|
+|FALSE|FALSE|FALSE|FALSE                          |FALSE                          |
+|FALSE|FALSE|TRUE |FALSE                          |FALSE                          |
+|FALSE|TRUE |FALSE|FALSE                          |FALSE                          |
+|FALSE|TRUE |TRUE |FALSE                          |FALSE                          |
+|TRUE |FALSE|FALSE|FALSE                          |FALSE                          |
+|TRUE |FALSE|TRUE |TRUE                           |TRUE                           |
+|TRUE |TRUE |FALSE|TRUE                           |TRUE                           |
+|TRUE |TRUE |TRUE |TRUE                           |TRUE                           |
+
 ### d
 
+| *A* | *B* | *C* |(*A* OR *B*) AND (*A* OR *C*)  |*A* OR (*B* AND *C*)           |
+|:---:|:---:|:---:|:-----------------------------:|:-----------------------------:|
+|FALSE|FALSE|FALSE|FALSE                          |FALSE                          |
+|FALSE|FALSE|TRUE |FALSE                          |FALSE                          |
+|FALSE|TRUE |FALSE|FALSE                          |FALSE                          |
+|FALSE|TRUE |TRUE |TRUE                           |TRUE                           |
+|TRUE |FALSE|FALSE|TRUE                           |TRUE                           |
+|TRUE |FALSE|TRUE |TRUE                           |TRUE                           |
+|TRUE |TRUE |FALSE|TRUE                           |TRUE                           |
+|TRUE |TRUE |TRUE |TRUE                           |TRUE                           |
+
 ## Question 4
 
 ### a
 
+| *A* | *B* |NOT (*A* AND *B*)  |NOT *A* OR NOT *B* |
+|:---:|:---:|:-----------------:|:-----------------:|
+|FALSE|FALSE|TRUE               |TRUE               |
+|FALSE|TRUE |FALSE              |FALSE              |
+|TRUE |FALSE|FALSE              |FALSE              |
+|TRUE |TRUE |FALSE              |FALSE              |
+
 ### b
 
+| type(*X*) = INT | *X* > 7   |(*X* = int AND *X* > 7) OR ( *X* = float AND *X* > 7)|(*X* = int or *X* = float ) and *X* > 7|
+|:---------------:|:---------:|:---------------------------------------------------:|:-------------------------------------:|
+|TRUE             |FALSE      |FALSE                                                |FALSE                                  |
+|TRUE             |TRUE       |TRUE                                                 |TRUE                                   |
+|FALSE            |FALSE      |FALSE                                                |FALSE                                  |
+|FASLE            |TRUE       |TRUE                                                 |TRUE                                   |
```

Comment:
Missing a column for `type(X) == float` (this has to be considered separately as e.g. `x` could be neither an int nor a float)